src/engraving/dom/score.cpp

```diff
@@ -169,6 +170,8 @@ Score::Score()
 
     m_shadowNote = new ShadowNote(this);
     m_shadowNote->setVisible(false);
+
+    setId(Uuid::createUuid());
```

Comment:
Can we set new ID only for new scores? (not for readed)

Reply:
Actually, I don't see a reason to do this only for new scores. 
Can you explain the purpose of this limitation?

Reply:
I don't like, what a score have one id and after read same score have different id. For example, if we want debugging or logging id of score, will be confused.